test/CMakeLists.txt

```diff
@@ -70,6 +70,7 @@ set(SRC
     nop_elimination.cpp
     op.cpp
     opset_pass/softmax_opset_pass.cpp
+    opset_pass/pad_opset1upgrade_pass.cpp
```

Comment:
I think we may want to keep all pad-related pass tests in one file. I would consider renaming this to `pad_opset_pass.cpp`

Reply:
Done